documentation/developer_updates.md

```diff
@@ -36,6 +36,30 @@ install a Puppetfile, because Bolt installs modules to a special directory that
 is not part of the configured modulepath.
 
 The new module management feature is available starting with **Bolt 2.30.0**. To
-try it out, opt in by updating your project. You can learn more about this
-feature and opting in at [Managing modules in Bolt
-projects](managing_modules.md).
+try it out, [create a
+project](bolt_installing_modules.md#create-a-bolt-project-with-pre-installed-modules)
+or [migrate an existing project](./projects.md#migrate-a-bolt-project).
+
+Here's a summary of what's changed:
```

Comment:
I think it'd be best to point out that `bolt project migrate` will do a lot of this for you towards the top here, but I'm not sure where to put it. Basically I think it's "Run `bolt project migrate` to use the new workflow. You shouldn't have to think about it, but here's what changed if you're interested"?